src/viam/sdk/resource/resource_manager.cpp

```diff
@@ -47,9 +47,9 @@ void ResourceManager::replace_all(
     this->resources_ = new_resources;
     this->short_names_ = new_short_names;
 
-    for (const auto& resource : resources) {
+    for (auto resource : resources) {
```

Comment:
I disagree with this one, looking at it. This works because it now copies the pair out, and then moves from it. That doesn't seem worthwhile vs just viewing the pair and copying the second in the call to `do_add`